### PR TITLE
Trino Summit Teaser post

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Trino Summit 2022 will be cute and legendary"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-tenth-birthday/creators.jpeg
+---
+
+Bun Bun is back and this year we have an exciting lineup of speakers. The range
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:9]:
```suggestion
Commander Bun Bun is back, and this year we have an exciting lineup of speakers. The range
```

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+of topics span from architectures like data mesh and data lakehouses, to running
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:10]:
```suggestion
of topics spans from architectures like data mesh and data lakehouses, to running
```

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Trino at scale with fault-tolerance and how to use `EXPLAIN`. This conference is
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:11]:
```suggestion
Trino at scale with fault-tolerant execution and how to use `EXPLAIN`. This conference is
```

maybe even break up the sentence

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+free and takes place on November 10th. The summit is in-person but also is
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:12]: The summit is a hybrid even for in-person and virtual attendance.

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+available to virtual attendees as well. Find out more details below!
+
+<!--more-->
+
+## Location
+
+This years Trino Summit will be hosted live at the Commonwealth Club in San 
+Francisco, CA. In-person registration is limited to 250 seats so make sure you
+register quickly before spots run out!
+
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:22]: Add the button from the other blog post here

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+## Announcing the first sessions and speakers
+
+1. Keynote: State of Trino
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:25]: Add an entry sentence that mentions that call for speakers closed and we have a full line up .. and here are a first few teasers

File: _posts/2022-09-16-trino-summit-2022-teaser.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+    Description: Hear the latest on the state of the open source Trino project.
+    Trino is the award-winning MPP SQL query engine. In this session, Trino and
+    Presto creators will discuss the latest features that have landed in the
+    last year, the roadmap for the year ahead, and community growth highlights.
+
+    * <img width="150" src="https://sessionize.com/image/40f9-0o0o0-Jd9Ri1YsJ7VyFniPcENpHv.jpg">
```
Comment (mosabua) [inline _posts/2022-09-16-trino-summit-2022-teaser.md:32]: Let not use images from sessionize .. either dont use them .. or get them into the site directly

Comment (mosabua) [reply _posts/2022-09-16-trino-summit-2022-teaser.md:32]: Just the names are perfectly fine as well btw. .. images are not really necessary

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+announce more session as they are accepted!
+
+### Keynote: State of Trino
+
+**Abstract:** Hear the latest on the state of the open source Trino project. Trino
+is the award-winning MPP SQL query engine. In this session, Trino and Presto
+creators will discuss the latest features that have landed in the last year, the
+roadmap for the year ahead, and community growth highlights.
+
+**Speakers:**
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:54]: do we need this subtitle .. its kinda obvious .. might work fine without

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+* Martin Traverso, Co-Creator of Trino and Chief Technology Officer, Starburst
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:56]: just use CTO

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+        
+* Dain Sundstrom, Co-Creator of Trino and Chief Technology Officer, Starburst
+        
+* David Phillips, Co-Creator of Trino and Chief Technology Officer, Starburst
+
+### Trino for Large Scale ETL at Lyft
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:62]:
```suggestion
### Trino for large scale ETL at Lyft
```

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+
+## Announcing the first sessions and speakers
+
+Here are all the speakers we have so far. Stay tuned for future blog posts as we
+announce more session as they are accepted!
+
+### Keynote: State of Trino
+
+**Abstract:** Hear the latest on the state of the open source Trino project. Trino
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:49]: I dont think we nee "Abstract"

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+during unexpected spikes, etc.
+              
+We'll share what we've done with our config tunings, large query/user
+identifications, autoscaling and fault tolerant features to execute Trino at
+such a scale. We'll also share our upcoming challenges and plans to move steps
+further with Trino adoption across the company.
+
+**Speaker:** Charles Song, Senior Software Engineer at Lyft
+    
+### Rewriting History: Migrating petabytes of data to Apache Iceberg using Trino
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:79]: history

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+evolving landscape, I will highlight recent contributions to Trino’s Iceberg
+integration that made our work possible while also illustrating how we designed
+our system to scale. Topics will include: what to consider when designing your
+migration strategy, how we optimized Trino’s write performance and how to
+recover from corrupt table states. Finally, I will compare the query performance
+of old and migrated datasets using Shopify’s datasets as benchmarks.
+
+**Speaker:** Mark Laforet, Senior Data Engineer at Shopify
+
+### Federating them all on Galaxy!
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:97]:
```suggestion
### Federating them all on Starburst Galaxy!
```

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+
+⚡ _Lightning talk_
+
+**Abstract:** You've federated them all on Trino, but to beat the elite four at
+Indigo Plateau, every data trainer needs help. In this talk, I will cover how
+Galaxy is the fastest path to query federation and cover a demo that trainers
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:103]:
```suggestion
Starburst Galaxy is the fastest path to query federation and cover a demo that trainers
```

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+can leverage Apache Airflow to orchestrate Trino queries into complex workflows
+that solve practical batch processing problems, all the while avoiding the use
+of repetitive, redundant data movement.
+
+**Speaker:** Philippe Gagnon, Solutions Architect at Astronomer
+
+## Conclusion
+
+Stay tuned for new developments in upcoming blog posts, don't forget to
+register, and always federate them all!
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:128]: make register a link here as well

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+
+## Announcing the first sessions and speakers
+
+Here are all the speakers we have so far. Stay tuned for future blog posts as we
+announce more session as they are accepted!
+
+### Keynote: State of Trino
+
+**Abstract:** Hear the latest on the state of the open source Trino project. Trino
+is the award-winning MPP SQL query engine. In this session, Trino and Presto
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-09-22-trino-summit-2022-teaser.md:50]: the InfoWorld BOSSIE awards among others like that.

**Comment by bitsondatadev** [comment]
Just added the link from the homepage banner.

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 17]
```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Trino Summit 2022 will be legendary"
+author: "Brian Olsen"
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:4]: we should add Dain as author .. he is in the video ;-)

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:6]: Nice catch to add that ..

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 19]
```diff
@@ -0,0 +1,128 @@
...
+---
+
+Commander Bun Bun is back and this year we have an exciting lineup of speakers.
+The range of topics spans from architectures like data mesh and data lakehouses,
+to running Trino at scale with fault-tolerant execution and query federation.
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:11]:
```suggestion
to running Trino at scale with fault-tolerant execution, and query federation.
```

File: _posts/2022-09-22-trino-summit-2022-teaser.md [hunk 20]
```diff
@@ -0,0 +1,128 @@
...
+our system to scale. Topics will include: what to consider when designing your
+migration strategy, how we optimized Trino’s write performance and how to
+recover from corrupt table states. Finally, I will compare the query performance
+of old and migrated datasets using Shopify’s datasets as benchmarks.
+
+Mark Laforet, Senior Data Engineer at Shopify
+
+### Federating them all on Starburst Galaxy!
+
+⚡ _Lightning talk_
```
Comment (mosabua) [inline _posts/2022-09-22-trino-summit-2022-teaser.md:98]: drop that .. unimportant detail ;-)

**Review by mosabua — CHANGES_REQUESTED** [review]
Minor nits.. lets ship it!

**Comment by mosabua** [comment]
Also .. we need to make the video public just before we merge...

**Review by mosabua — APPROVED** [review]
Federate them all!

**Pull request merged**